css/properties/mask.json

```diff
@@ -35,10 +35,17 @@
                 "version_removed": "79"
               }
             ],
-            "firefox": {
-              "version_added": "2",
-              "notes": "From Firefox 10, the default color space when handling masks is sRGB. Previously, the default and only supported color space was linear RGB. This changes the appearance of mask effects, but brings Firefox into compliance with the second edition of the SVG 1.1 specification."
-            },
+            "firefox": [
+              {
+                "version_added": "53",
+                "notes": "Added support for CSS Masking Module Level 1, including the <code>mask</code> shorthand, and <code>mask-image</code>, <code>mask-repeat</code>, <code>mask-size</code>, etc."
```

Comment:
I don't think we should have this note, which is about mask support more generally. This entry is about just the `mask` shorthand, and that's supported it sounds like?

Reply:
I was trying to convey the idea that the `mask` shorthand effectively changed radically when CSS Masking Module Level 1 was created.

The problem is that the question “is the `mask` shorthand property supported?” does not mean much by itself, and is misleading. Do we mean:

1. Support for the `mask: url(#some_id)` feature where `#some_id` is a SVG element and this style is applied to a SVG element?
2. Support for the `mask: url(#some_id)` feature where `#some_id` is a SVG element and this style is applied to a HTML element?
3. Support for CSS Masking Module Level 1 for HTML elements?

My take was that “support” should be CSS Masking Module Level 1, and the earlier SVG feature should be marked as partial support.

Happy to do it differently if there's a better convention for when a feature was completely overhauled in standards while keeping the same name.

Reply:
> This entry is about just the mask shorthand

My main concern is that it's used in this MDN page:
https://developer.mozilla.org/en-US/docs/Web/CSS/mask

That page is about the shorthand but describes the CSS Masking Module Level 1 feature set in some details, then the compatibility table says it's supported since Firefox 2. For any reasonable reader, this reads as “Firefox 2 supports the features shown in this page”, which isn't true.

Reply:
Sorry, I overlooked this in my GitHub notifications :)

The `mask` property is more complicated than I thought then. Would a note like this be accurate?

> Only supports `mask: url(#some_id)` where `#some_id` is a reference to an SVG element.

That would exclude both examples in https://developer.mozilla.org/en-US/docs/Web/CSS/mask#examples, but was just `mask: url(resources.svg)` supported earlier on in Firefox?

Reply:
I'm not sure what syntax Firefox <52 supported exactly.

I tried downloading Firefox 50 but it doesn't run on a recent macOS.

So I'm searching the Web for content published between 2010 and early 2017 (excluding the Firefox 52 release and later content) to see what was actually supported.

Here's a StackOverflow answer that says that Firefox only supports `mask: url(file.svg#mask_id)`:
https://stackoverflow.com/a/38652557

I think this also included `mask: url(#mask_id)` (looking into the current SVG document or the current HTML document).

And a 2016 article showing the CSS Masking Module Level 1 usage in WebKit and Chrome (without the `-webkit-` prefix, but the Codepen demos do use the prefix), and shows Firefox only supporting `mask: url(file.svg#mask_id)` with an URL to a SVG `<mask>` element:
https://www.sitepoint.com/masking-in-the-browser-with-css-and-svg/